src/v2/components/settings/SqliteConsole.tsx

```
@@ -0,0 +1,110 @@
+              >
+                <Play className="h-4 w-4 mr-1" />
+                {t('debug.sqliteConsole.execute') || '実行'}
+              </Button>
+            </div>
+          </div>
+
+          <div className="flex-1 overflow-hidden flex flex-col gap-2">
+            <label className="text-sm font-medium text-gray-700 dark:text-gray-300">
+              {t('debug.sqliteConsole.resultLabel') || 'Result'}
+            </label>
+            <div className="flex-1 overflow-auto">
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)